src/functional/__snapshots__/markdown-empty.spec.ts.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`createMarkdown single project outputs correct changelog 1`] = `""`;
```

Comment:
I don't understand this. why are all the snapshots empty?

Reply:
The cases are for empty messages. 

Earlier the renderer used to send empty line breaks. #118 . The fix on this bug is to avoid that.

Reply:
@Turbo87  Lemme know if I need to change anything over this.